examples/kubernetes/dynamic_provisioning/specs/storageclass.yaml

```diff
@@ -6,3 +6,5 @@ provisioner: fsx.csi.aws.com
 parameters:
   subnetId: subnet-056da83524edbe641
   securityGroupIds: sg-086f61ea73388fb6b
+mountOptions:
+  - flock
```

Comment:
I think if `flock` is only required under certain situation. Could you create a new `MountOption` example for just this use case? I suspect user most of the time just copy paste examples and assume it as best practices, which might be not true for using `flock` option in dynamic provisioning generally.

Same comment in other examples.

Reply:
I think flock a reasonable default/best practice because we put it in our docs as the default and tell users to remove it if they don't want it: 
"flock – mounts your file system with file locking enabled. If you do not want file locking enable, use the command without flock." https://docs.aws.amazon.com/fsx/latest/LustreGuide/mounting-ec2-instance.html. So in the same way, I think it's up to the user to remove flock from the StorageClass if they don't want it.

Like, there is nothing preventing a user from (inadvertently) mounting the same PVC in different pods (maybe they scale up their deployment) so flock seems to be the "safe" option